afs-spring-server/src/main/java/com/powsybl/afs/server/StorageServer.java

```diff
@@ -0,0 +1,605 @@
+        return ok();
+    }
+
+    @PostMapping(value = "fileSystems/{fileSystemName}/nodes/{nodeId}/timeSeries/string/{version}/{timeSeriesName}", consumes = MediaType.APPLICATION_OCTET_STREAM_VALUE)
+    @ApiOperation(value = "")
+    @ApiResponses(value = {@ApiResponse(code = 200, message = ""), @ApiResponse(code = 500, message = "Error")})
+    public ResponseEntity<String> addStringTimeSeriesData(@ApiParam(value = "File system name") @PathVariable("fileSystemName") String fileSystemName,
+                                                          @ApiParam(value = "Node ID") @PathVariable("nodeId") String nodeId,
+                                                          @ApiParam(value = "Version") @PathVariable("version") int version,
+                                                          @ApiParam(value = "Time Series Name") @PathVariable("timeSeriesName") String timeSeriesName,
+                                                          @ApiParam(value = "List string array chunkFile system name") @RequestBody List<StringDataChunk> chunks) {
+        AppStorage storage = appDataWrapper.getStorage(fileSystemName);
+        storage.addStringTimeSeriesData(nodeId, version, timeSeriesName, chunks);
+        return ok();
+    }
+
+    @PutMapping(value = "fileSystems/{fileSystemName}/nodes/{nodeId}/metadata", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Update node's metadata")
+    @ApiResponses(value = {@ApiResponse(code = 200, message = ""), @ApiResponse(code = 500, message = "Error")})
+    public ResponseEntity<String> setMetadata(@ApiParam(value = "File system name") @PathVariable("fileSystemName") String fileSystemName,
+                                              @ApiParam(value = "Node ID") @PathVariable("nodeId") String nodeId,
+                                              @ApiParam(value = "Node Meta Data") @RequestBody NodeGenericMetadata nodeMetadata) {
+        AppStorage storage = appDataWrapper.getStorage(fileSystemName);
+        storage.setMetadata(nodeId, nodeMetadata);
+        return ok();
+    }
+
+    private static <T> ResponseEntity<T> ok() {
+        return ResponseEntity.ok().build();
+    }
+
+    private static <T> ResponseEntity<T> noContent() {
+        return ResponseEntity.noContent().build();
+    }
+
+    private static <T> ResponseEntity<T> ok(T body) {
+        return ResponseEntity.ok(body);
+    }
+
+    private static <T> ResponseEntity<T> okIfPresent(Optional<T> body) {
+        return body
+                .map(StorageServer::ok)
+                .orElseGet(StorageServer::noContent);
+    }
+
+    private static void logInfo(String message, Object... params) {
+        if (LOGGER.isInfoEnabled()) {
+            Object[] objects = Arrays.stream(params)
+                    .map(StorageServer::encode)
+                    .toArray();
+            LOGGER.info(message, objects);
+        }
+    }
+
+    private static Object encode(Object input) {
```

Comment:
maybe add a comment explaining why we remove these kind of whitespace ? What happens if the input is not a string, so it uses its toString() method to be formatted in the message, but the toString() produce this kind of whitespace ? Some comments explaining all this would make it more understandable

Reply:
Indeed, that method does not seem useful to me:
actually it can even make the logs confusing, since for instance we might log modified names or identifiers which will not help debugging.

Also, I would propose to make those start/cancel logs debug level instead of info : would it be OK @PratheepDev , @pl-buiquang .

Reply:
This was part of a sonar fix (taken from the AppStorageServer code) : https://github.com/powsybl/powsybl-afs/pull/48/commits/76ab4e856c29c1304c83a0f3c23dd439c6c5f428

Why would you want to set the start/cancel logs to debug ? this is not really low level api action so info seems right to me